addons/binding/org.openhab.binding.mihome/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="mihome"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>Energenie Mi|Home Binding</name>
+    <description>This is the binding for the Energenie Mi|Home house home automation solution</description>
+    <author>Musala Soft JSC</author>
```

Comment:
It seems to be OK for me - https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.coolmasternet/ESH-INF/binding/binding.xml